lib/routes/password.js

```diff
@@ -348,6 +351,11 @@ module.exports = function (
               )
               .then(
                 function() {
+                  return request.emitMetricsEvent('password.forgot.send_code')
```

Comment:
Were you thinking like `password.forgot.send_code.start`, after validating metrics, and `password.forgot.send_code.completed` here instead of plain `password.forgot.send_code`?